snapcraft/extensions/gnome.py

```diff
@@ -0,0 +1,212 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2022 Canonical Ltd
```

Comment:
For new files we're using a different copyright line format.

Reply:
Good catch